deployer/README.md

```diff
@@ -372,6 +373,71 @@ for the main cluster.
 
 It is recommended to have one curator for each Elasticsearch cluster.
 
+### About the Deployer generated secrets
+
+Part of the installation process that is done by the logging deployer is to generate
+certificates and keys and make them available to the logging components by means
+of secrets.
+
+The secrets that the components make use of are:
+
+    logging-curator
+    logging-curator-ops
+    logging-elasticsearch
+    logging-fluentd
+    logging-kibana
+    logging-kibana-proxy
+
+#### logging-curator, logging-kibana, logging-fluentd
+
+These three components all have a `ca`, `key` and `cert` entries.  These are what
+are used for mutual TLS communication with Elasticsearch.
+  1. `ca` contains the certificate that Elasticsearch uses.
+  2. `key` is the generated key specific to that component.
+  3. `cert` is the certificate created using `key`.
+
+#### logging-kibana-proxy
+
+The Kibana proxy container is used to communication with clients outside of the
+cluster.  It contains `oauth-secret`, `server-cert`, `server-key`, `server-tls.json`
+and `session-secret`.
+  1. `oauth-secret` is used to communicate with the oauthclient created as part of
+the aggregated logging installation to redirect requests for authentication with
+the OpenShift console.
+  2. `server-cert` is the cert used to communicate back to Kibana
+  3. `server-key` is the key used to communicate back to Kibana
+  4. `server-tls.json` is the proxy TLS configuration file
+  5. `session-secret` contains the generated proxy session
```

Comment:
it's for securing the user cookie which contains their auth token (once they have one).